spanner/api/Spanner.Samples.Tests/CancelBackupOperationTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class CancelBackupOperationTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public CancelBackupOperationTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public void TestCancelBackupOperation()
+    {
+        CancelBackupOperationSample cancelBackupOperationSample = new CancelBackupOperationSample();
+        var result = cancelBackupOperationSample.CancelBackupOperation(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.DatabaseId, _spannerFixture.ToBeCancelledBackupId);
+        Assert.True(result);
```

Comment:
Is this not going to be flaky? Can it be that from time to time the backup is actually created before we can cancel? Realistically, how long does it takes a backup to be created?

Reply:
yes it going to flaky. changed the logic now list backups call and check the backup with specific backupId